Updated docu for VM controls in Flex (#55)
SAP-docs/sapui5: docs/05_Developing_Apps/enabling-ui-adaptation-other-things-to-consider-de9fd55.md

All you have to do is add this control to an appropriate location in your app, and assign the desired UI container as a target in a `for` association.

> Please note that retroactively introducing a variant management control after a key user has already created adaptations for your application is not supported. Such an introduction would lead to adaptations being applied in a non-defined order, making the state of the UI unpredictable. 

Reviewer: ```suggestion
Please note that it is not supported to retroactively introduce a variant management control after a key user has already created adaptations for your application. This would lead to adaptations being applied in a non-defined order, making the state of the UI unpredictable.
```
